docs/DevGuide/GitHubActions.md

```diff
@@ -165,24 +177,10 @@ specifying the environment variables:
 \code{.sh}
 CCACHE_COMPILERCHECK=content
 CCACHE_EXTRAFILES="${SPECTRE_SOURCE_DIR}/tools/SpectrePch.hpp"
-CCACHE_IGNOREHEADERS="${SPECTRE_BUILD_DIR}/SpectrePch.hpp:${SPECTRE_BUILD_DIR}/SpectrePch.hpp.gch"
+CCACHE_IGNOREHEADERS=\
```

Comment:
the `$` in these environment variables are being rendered as `\f$`

Reply:
See #4036

Reply:
This might render correctly if you replace `\code` with three backticks. Either way, I'll take a look.